.docker/experimental/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# moveit/moveit:kinetic-experimental
+# Based on a moveit source install, adds the mongo driver and the warehouse packages
+
+FROM moveit/moveit:kinetic-source
```

Comment:
I thought it'd be cleaner to have an overlay? Happy to moveit to the same workspace